mitmproxy/http.py

```diff
@@ -9,6 +9,7 @@
 
 HTTPRequest = http.Request
 HTTPResponse = http.Response
+HTTPMessage = http.Message
```

Comment:
@mhils didn't we try to get rid of these (now empty) wrapper classes?

Reply:
Yes, not sure what I was thinking here. 👍